src/SymlinkFolderFileContents.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\Deploy;
+
+use Robo\Collection\CollectionBuilder;
+use Robo\Contract\BuilderAwareInterface;
+use Robo\Task\BaseTask;
+use Robo\Task\Filesystem\FilesystemStack;
+use Symfony\Component\Finder\Finder;
+
+class SymlinkFolderFileContents extends BaseTask implements BuilderAwareInterface
```

Comment:
So the goal is to call a robo command via an ssh command to do the deploy. Can instructions for this be added in the README already? :)